engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -50,6 +51,7 @@ class SingleWorkflowRunnerActor(source: WorkflowSourceFilesCollection,
   override lazy val jobStoreActor = context.actorOf(EmptyJobStoreActor.props, "JobStoreActor")
   override lazy val subWorkflowStoreActor = context.actorOf(EmptySubWorkflowStoreActor.props, "SubWorkflowStoreActor")
 
+  log.info("{}: Version {}", Tag, VersionUtil.getVersion("cromwell-engine"))
```

Comment:
What's `Tag` and why doesn't the server mode get one?

Reply:
https://github.com/broadinstitute/cromwell/blob/63637a61d138f87de99f4129f8b25a5a18d8088d/engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala#L258

Logging for Single vs. Server is a complete mystery to me. This syntax was just copied from other existing logging. Ex:

https://github.com/broadinstitute/cromwell/blob/63637a61d138f87de99f4129f8b25a5a18d8088d/engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala#L57

https://github.com/broadinstitute/cromwell/blob/63637a61d138f87de99f4129f8b25a5a18d8088d/engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala#L181

Reply:
🤷‍♂️